Tests/webidl2swiftTests/webidl2swiftTests.swift

```diff
@@ -5,7 +5,7 @@
 import XCTest
 import ArgumentParser
 import Commands
-@testable import webidl2swift
+@testable import Commands
```

Comment:
Why this change?

Reply:
Latest Xcode complains about `webidl2swift` being an executable module, it says such modules can no longer be imported in test targets. So I'm importing `Commands`, which is a library target.

Reply:
Ok, cool!